app/models/energy/hvac.py

```diff
@@ -0,0 +1,45 @@
+"""HVAC Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+
+
+class EconomizerType(str, Enum):
+    no_economizer = 'NoEconomizer'
+    differential_dry_bulb = 'DifferentialDryBulb'
+    differential_enthalpy = 'DifferentialEnthalpy'
+
+
+class IdealAirSystem(BaseModel):
+    """ Provides a model for an ideal HVAC system."""
+    type: Enum('IdealAirSystem', {'type': 'IdealAirSystem'})
+
+    heating_limit: Union[float, str] = Schema(
+        'autocalculate',
```

Comment:
I think we might need a validator to make sure that someone doesn't add text other than "autocalculate" since any other strings are not acceptable by E+.